Asterix/optics/propagation_functions.py

```diff
@@ -3,21 +3,54 @@
 
 
 def mft(image,
-        real_dim_input,
-        dim_output,
-        nbres,
+        real_dim_input=4,
+        dim_output=4,
+        nbres=1,
```

Comment:
yes, because I'm not sure it will be working at all for a 1 pixel array. Somehting that can be done is to do none, none, none to these optional argument and if you don't fill them ,the MFT does an fft :
real_dim_input = dim_imput
dim_output = dim_imput
nbres = 2 
?

Reply:
Nah that's fine. I was just curious.